lib/src/main/java/com/futurewei/alcor/common/db/ignite/IgniteTransaction.java

```diff
@@ -33,55 +31,48 @@
 public class IgniteTransaction implements Transaction {
     private static final Logger logger = LoggerFactory.getLogger();
 
-    private IgniteClient igniteClient;
-    private ClientTransaction clientTransaction;
+    private final Ignite client;
```

Comment:
@Gzure Should Ignitetransaction also support IgniteClient based on the setting?

This is non-blocking. If we do need that, we can do it next PR.

Reply:
> @Gzure Should Ignitetransaction also support IgniteClient based on the setting?
> 
> This is non-blocking. If we do need that, we can do it next PR.

I had added a IgniteClientTransaction which according to Ignite thin client